templates/nextjs-typescript/Dockerfile

```diff
@@ -12,10 +12,10 @@ WORKDIR /app
 
 RUN yarn import || echo "lockfile already exists"
 
-# Install dependencies and delete the cache
-RUN set -ex; yarn install --frozen-lockfile --cache-folder /tmp/.cache; rm -rf /tmp/.cache;
 # Copy sources
 COPY . .
+# Install dependencies and delete the cache
```

Comment:
The error was: The command '/bin/sh -c yarn build' returned a non-zero code: 126

If we take a look at this official docker example below, they are using multiple stages to do the deps and build, I can rework it and see.

https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile

Reply:
> 
> 
> This isn't ideal, it'll make this docker layer download dependencies every time there is a change to the source, even when no dependencies are changed. What was the error on Windows?

The error was a 126 error on yarn build. I have updated the docker image to set permissions for next instead of the previous change.